public/components/datasources/components/__tests__/manage_datasource_table.test.tsx

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Not sure how this got pulled in, should be on main already, can you double check/merge